install.py

```diff
@@ -124,6 +127,9 @@ def main():
     prefix = "the relative path to the installation directory"
     parser.add_argument('--prefix', help=prefix, default=localdir)
 
+    config_only = 'only configure the package, do not build or install'
```

Comment:
only runs cmake

On Thu, May 28, 2015 at 11:17 AM, opotowsky <notifications@github.com>
wrote:

> In install.py
> <https://github.com/cyclus/cyclus/pull/1173#discussion_r31250848>:
>
> > @@ -124,6 +127,9 @@ def main():
> >      prefix = "the relative path to the installation directory"
> >      parser.add_argument('--prefix', help=prefix, default=localdir)
> >
> > +    config_only = 'only configure the package, do not build or install'
>
> @gidden <https://github.com/gidden>, a quick search did not teach me what
> exactly this means. this works for me, I'm just trying to understand it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1173/files#r31250848>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192